### PR TITLE
login page

### PR DESCRIPTION
**Pull request by Gifftybabe**

### CODE REVIEW

**Review by Favvie — DISMISSED** [review]
There are two background images in your screen.

Please refactor to the form component in shadcn

File: client/Components/Registration.tsx [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
+    const {handleSubmit, control, formState: {errors}} = form;
+
+    const onSubmit: SubmitHandler<TSignupFormSchema> = (data) => {
+      console.log(data)
+    }
+
+    return (
+<Form {...form}>
+   <form className="space-y-6 w-full" onSubmit={handleSubmit(onSubmit)}>
+          {/* <div className="flex items-center space-x-4">
```
Comment (Favvie) [inline client/Components/Registration.tsx]: Please remove the unused code

File: client/Components/Registration.tsx [hunk 2]
```diff
@@ -1,46 +1,51 @@
...
+  location: yup.string().required("Please  enter your location"),
+  firstName: yup.string().required("Enter your first name"),
+  lastName: yup.string().required("Enter your last name"),
+  phoneNumber: yup.string().test("is-number", "Please enter a number", (value) => {
+    if(!value) return true;
 
-        <div className="flex h-screen items-center justify-center z-50">
-          <div className="flex flex-col items-center space-y-2">
-            <div className="flex items-center">
+    return !isNaN(parseFloat(value)) && isFinite(+value))
```
Comment (Favvie) [inline client/Components/Registration.tsx]: remove the bracket, and add a ; it is causing your code to break

Comment (Belloabraham121) [reply client/Components/Registration.tsx]: WhatsApp Image 2024-10-14 at 15 02 50_7b2e6006
WhatsApp Image 2024-10-14 at 15 03 18_27a670c1

**Comment by Gifftybabe** [comment]
> There are two background images in your screen.
> 
> Please refactor to the form component in sha

One of the images removed already

**Review by obah — APPROVED** [review]

**Pull request merged**